packages/react-ui-components/src/SelectBox_Option_MultiLineWithThumbnail/selectBox_Option_MultiLineWithThumbnail.js

```diff
@@ -42,9 +42,9 @@ class SelectBox_Option_MultiLineWithThumbnail extends PureComponent {
         return (
             <ListPreviewElement {...rest} icon={icon} className={finalClassNames}>
                 {Boolean(imageUri) && <img src={imageUri} alt={label} className={theme.multiLineWithThumbnail__image}/>}
-                <span>{label}</span>
-                {Boolean(secondaryLabel) && <span className={theme.multiLineWithThumbnail__secondaryLabel}>{secondaryLabel}</span>}
-                {Boolean(tertiaryLabel) && <span className={theme.multiLineWithThumbnail__tertiaryLabel}>{tertiaryLabel}</span>}
+                <span title={label}>{label}</span>
```

Comment:
How does it help to have a title identical to the label?

Reply:
It helps on long strings, especially for page titles and URLs